src/shell/Shell.cpp

```diff
@@ -634,6 +513,390 @@ static bool evalScript(ContextRef* context, StringRef* source, StringRef* srcNam
     return result;
 }
 
+struct WorkerThreadData {
+    std::thread thread;
```

Comment:
This `thread` member seems needless too.
each thread is allocated in `workerThreads` vector in the below
```c++
std::vector<std::pair<std::thread, WorkerThreadData>> workerThreads;
```

Reply:
I removed it!